effectsize_example.Rmd

```diff
@@ -85,7 +100,7 @@ library(ggstance)   # for geom_pointrangeh()
 
 ```{r boilerplate, include = FALSE}
 set.seed(12)
-format_num <- function(...) format(..., digits = 2)
+format_num <- function(x, k = 2) format(round(x, k), nsmall = k)
```

Comment:
Updated the `format_num` function in the latest commit. I also had to manually determine significant digits at each place based on the value and unit. (For example, I had to assume that measuring the reaction time in ms is adequate without decimal places.)

My first concern on this would be that the readers have to take special care in using the code, in the reporting part. May be we should capture this discussion in the FAQ. If you think so, add thumb-up to this comment. Once we have two thumb-ups, please create an issue for this.

(continued below)